src/services/expensesService.js

```diff
@@ -0,0 +1,69 @@
+const { data } = require('../data');
+
+const getAllExpenses = () => {
+  return data.expenses;
+};
+
+const getById = (id) => {
+  return data.expenses.find((exp) => exp.id === parseInt(id));
```

Comment:
Bug: The `id` parameter is parsed to integer using `parseInt(id)`. This could potentially lead to unexpected results if the `id` is not a number. It's better to ensure that `id` is a number before calling this function or handle this case properly.